src/taskManager/Taskable.java

```diff
@@ -0,0 +1,6 @@
+package taskManager;
+
+public interface Taskable {
+    void changeStatus(String newStatus);
+    void editTask(String title, String description, String priority);
```

Comment:
This method looks inconvenient - there's no way to change only one field in the task, a user will have to provide all 3 fields. So it's not user-friendly. Also, validation becomes less user-friendly - a user might provide an invalid value via the console UI for one of the fields and won't be able to fix it without entering all 3 values again.

Reply:
An improvement here could be achieved by
1) creating separate change/set method for each field
2) either returning a boolean value from the method to indicate operation success or (more preferable) to declare that these methods might throw validation exception

Reply:
Thank you for suggestions! I'll keep them in mind :)
The methods were designed this way for educational purposes, and I can't change their signatures.
 I added a "return value" to indicate whether the operation was successful.